content_scripts/VirtualCamera.js

```diff
@@ -760,7 +756,8 @@ import * as THREE from '../../thirdPartyCode/three/three.module.js';
                 return; // don't animate the matrix with an infinite level of precision, stop when it gets very close to destination
             }
 
-            let shouldSmoothCamera = !this.isRendering2DVideo() && !this.zoomOutTransition;
+            // disables smoothing while following, to provide a tighter sync with the followed element
+            let shouldSmoothCamera = !this.followingState.active && !this.zoomOutTransition;
```

Comment:
In general I feel like smoothing in 3rd person perspectives looks better

Reply:
Good point, I'll bring that back